app/models/insights_hit.rb

```diff
@@ -0,0 +1,2 @@
+class InsightsHit < ApplicationRecord
```

Comment:
Shouldn't this define the hit-to-facet association? Just for the sake of completeness